### DISCUSSION TITLE
Downloading images

### DISCUSSION THREAD

**Discussion by elhennig** [QuantumEntangledAndy/neolink#76] [General]
Is there a way to download images from the sd card of the reolink cam? I use the Argus PT and I am looking for a solution to batch download all images from time to time

**Comment by QuantumEntangledAndy** [comment]
Not yet. It's on the todo list, but I'm focusing on getting the stream smooth first

**Reply by QuantumEntangledAndy** [reply]
If you can code PRs are welcome

**Reply by elhennig** [reply]
I would like to help, but your are coding in Rust, right? I am absolutely not familiar with that, sorry! 

Besides that time is a very limiting factor, too.

**Reply by QuantumEntangledAndy** [reply]
Yes time is often the limiting factor. Sometimes I don't know how i find any to code at all. 

Anyways you'll have to wait until I can finish up my todo list before I tackle this one.
